.github/workflows/cli-regression.yml

```diff
@@ -16,8 +16,8 @@ jobs:
   cli-regression-test:
     runs-on: ubuntu-latest
 
-    permissions:
-      actions: read
+    # permissions:
+    #   actions: read
 
```

Comment:
Originally added for increased security around `GITHUB_TOKEN` permissions, but I believe this is better handled at the repo settings level. 

<img width="500" alt="Screen Shot 2022-09-13 at 1 44 14 AM" src="https://user-images.githubusercontent.com/63436329/189855736-96d8f01c-b063-4550-9829-25adab79920a.png">